docs/content/version-reporting.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Anonymous Version Reporting
+kind: operations
+weight: 50
+---
+
+This document provides details about OPA's anonymous version reporting feature.
+
+## Overview
+
+OPA supports anonymous version reporting wherein OPA would reach out to an external service and share its version 
+information. This feature is only applicable to the `opa run` and `opa version` commands.
+
+In case of the `opa run` command, this feature will be **ON by-default** and can be easily disabled by specifying
+the `--incognito` flag. When OPA is started in either the `server` or `repl` mode, OPA would call the external service
+on a best-effort basis and share the version it's running. The time taken to execute the remote call and process the
+subsequent response from the external service would not delay OPA's start-up.
+
+In case of the `opa version` command, this feature can be enabled by specifying the `--check` or `-c` flag.
+
+## External Service
+
+OPA uploads its version information at [telemetry.openpolicyagent.org](https://telemetry.openpolicyagent.org). This is
```

Comment:
It's probably minimal work to support make the URL configurable (e.g., we could even special case this and just allow an environment variable to set the value, e.g., `OPA_TELEMETRY_SERVICE_URL=https://acmecorp.org/opa-telemetry`).

Reply:
Added the env var `OPA_TELEMETRY_SERVICE_URL`

Reply:
It would be good if this page did not return a 404. Perhaps make it run a text/plain response w/ the latest version, download URL, release notes URL, a short paragraph explaining what the telemetry service does, and a link off back to the privacy page for more details.